.github/shared/test/exec.test.js

```diff
@@ -56,6 +56,21 @@ describe("execNpm", () => {
   });
 });
 
+describe("execNpmExec", () => {
+  // A command run in the context of "npm exec --no -- ___" needs to call
+  // something referenced in package.json. In this case, js-yaml is present
+  // so it is used.
+  it("runs js-yaml", async () => {
+    await expect(
+      execNpmExec(["js-yaml", "--version"]),
```

Comment:
```suggestion
      execNpmExec(["js-yaml", "--version"], options),
```

Please update all tests in this file to pass an options bag including a ConsoleLogger, like this one:

```
const options = { logger: new ConsoleLogger(/*debug*/ true) };
```

https://github.com/Azure/azure-rest-api-specs/blob/main/.github/shared/test/readme.test.js#L9